roles/network_plugin/calico/templates/calico-node.yml.j2

```diff
@@ -77,8 +77,8 @@ spec:
 #            # Configure the IP Pool from which Pod IPs will be chosen.
 #            - name: CALICO_IPV4POOL_CIDR
 #              value: "192.168.0.0/16"
-#            - name: CALICO_IPV4POOL_IPIP
```

Comment:
is `CALICO_IPV4POOL_IPIP` per default? Currently it's out-commented but now you actually set it to `always`

Reply:
It should be Off, as default is Off